index.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-var crypto = require('crypto');
+const crypto = require('crypto');
```

Comment:
this is a breaking change for pre-const engines.

Reply:
Yeah an oversight on my part.
Thanks for the response.